src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -869,6 +869,7 @@ paths:
           schema:
             type: string
             enum:
+              - steward
```

Comment:
Is the idea here that adding `admin` users would only happen by updating the admin users group in SAM?

Reply:
Exactly.  The only way to become an admin is to be a part of the admin terra group